jason.json

```diff
@@ -0,0 +1,28 @@
+{
+    "response": [
+        {
+            "maintainer": "Mert Özkan (inits)",
+            "oem": "Xiaomi",
+            "device": "Mi Note 3",
+            "filename": "crDroidAndroid-15.0-20250124-jason-v11.1.zip",
+            "download": "https://github.com/mrtozkn/jason_crdroid/releases/download/20250124/crDroidAndroid-15.0-20250124-jason-v11.1.zip",
+            "timestamp": 1737667156,
+            "md5": "5346b80380ce2a005ac9cad12bd156f0",
+            "sha256": "51b554233cf5faef23a87b1ade78d38ff50317bfb6829b9ed7a8202de0d32f99",
+            "size": 1175760559,
+            "version": "11.1",
+            "buildtype": "Monthly",
+            "forum": "https://xdaforums.com/t/rom-15-0-unofficial-crdroid-android-v11-1-for-xiaomi-mi-note-3.4715048/",
+            "gapps": "https://sourceforge.net/projects/nikgapps/files/Elite-Releases/crdroid-official/Android-15/31-Dec-2024/NikGapps-crdroid-official-arm64-15-20241231-signed.zip/download",
```

Comment:
link to nikgapps site not specific gapps date would be better
https://nikgapps.com/crdroid-official